test/backend_test.in.cpp

```diff
@@ -7371,3 +7376,74 @@ NGRAPH_TEST(${BACKEND_NAME}, quantize_dynamic_offset)
     EXPECT_EQ((vector<output_c_type>{1, 1, 2, 3, 3, 3, 4, 5, 5, 5, 6, 7}),
               read_vector<output_c_type>(y));
 }
+
+#if defined(BACKEND_TEST_CPU) || defined(BACKEND_TEST_INTERPRETER)
```

Comment:
If you move batch_mat_mul to fused ops and provide a decomposition, then you wouldn't need to make this exemption.